mindsdb/integrations/handlers/rag_handler/requirements.txt

```diff
@@ -0,0 +1,15 @@
+langchain==0.0.267
+SQLAlchemy
+duckdb==0.7.1
+chromadb==0.3.25
+requests==2.28.0
+openai>=0.27.0
+pydantic>=1.10.8
+sentence_transformers==2.2.2
+markupsafe == 2.0.1
+faiss-cpu>=1.7.4 #todo check support for 'faiss-gpu'
+datasets>=2.14.4
+nltk>=3.8.1
+rouge-score>=0.1.2
+html2text
+writerai==0.25.3
```

Comment:
Got this while installing reqs:

```
ERROR: Cannot install -r ./mindsdb/integrations/handlers/rag_handler/requirements.txt (line 15) and -r ./mindsdb/integrations/handlers/rag_handler/requirements.txt (line 5) because these package versions have conflicting dependencies.

The conflict is caused by:
    requests 2.28.0 depends on charset-normalizer~=2.0.0
    writerai 0.25.3 depends on charset-normalizer>=2.1.1
```

Reply:
hmm, let me try to reproduce

Reply:
all works on my end with 3.10

Reply:
Ok, this is due to my env not being a fresh install and having some dependencies installed previously. 

@tomhuds note this is one anecdotal example of something #7027 would solve.